docs/topics/extensions.md

```diff
@@ -179,7 +179,7 @@ The following is an example of the template.json file.
    },
    "resources": [
     {
-      "apiVersion": "[parameters('apiVersionStorage')]",
+      "apiVersion": "[parameters('apiVersionDeployments')]",

```

Comment:
I think all the instances of `parameters('apiVersionDeployments')` need to be changed to `variables('apiVersionDeployments')`, right? I assume the `parameters('apiVersionStorage')` are docs cruft that wasn't cleaned up when some unit of params/vars rationalization work was completed.

Reply:
good point, I was doing some automated search and replace. I'll change these to variables.

Reply:
actually no sorry, they _are_ parameters. This is a deployment template with its own parameters defined.

Reply:
https://github.com/Azure/aks-engine/pull/1133/files#diff-433cd94fef7d8fc09ee6e9ca8379b874R138